server/bin/state/config-satellite/pbench-server.cfg

```diff
@@ -11,6 +11,7 @@ pbench-top-dir = %(unittest-dir)s/pbench-satellite
 pbench-local-dir = %(unittest-dir)s/pbench-satellite-local
 # A satellite just unpacks and syncs
 dispatch-states = TO-UNPACK, TO-SYNC
+unpacked-states = UNPACKED
```

Comment:
UNPACKED is the standard link desitnation for pbench-unpack-tarballs, I think `unpacked-states` should be all the others that depend on the unpacked tarball being there: TO-INDEX etc.

Reply:
Good catch.

Reply:
Fixed.